src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -97,4 +100,33 @@ private void setMetadata(Feature[] features, String jsonFile, int jsonFileNo) {
             feature.setMetaData(jsonFile, jsonFileNo, configuration);
         }
     }
+
+    /**
+     * Parses passed properties files and adds metadata to overview-features page.
+     *
+     * @param propertiesFiles
+     *            property files to read
+     */
+    public void parsePropertiesFiles(List<String> propertiesFiles) {
+        if (propertiesFiles != null) {
+            for (String propertyFile : propertiesFiles) {
+                processClassificationFile(propertyFile);
+            }
+        }
+    }
+
+    private void processClassificationFile(String file){
+        try {
+            PropertiesConfiguration config = new PropertiesConfiguration(file);
+            Iterator<String> keys = config.getKeys();
+            while (keys.hasNext()) {
```

Comment:
does it support for-each loop so then iterator is not needed?

Reply:
Unfortunately 

```
config.getKeys();
```
returns an iterator. https://commons.apache.org/proper/commons-configuration/apidocs/org/apache/commons/configuration2/AbstractConfiguration.html#getKeys--

as support for 7 & 8 is needed that rules out use of streams & forEachRemaining. Other alternatives could be use of (which is adding a new library to the code)
```
Guava or org.apache.commons.collections.IteratorUtils
```
& converting iterator to iterable list, Thoughts or leave as it is?

Reply:
sure